### ISSUE TITLE
Explanation of all settings in sample project menu

### ISSUE THREAD

**Issue by Jversen** [Azure/azure-object-anchors#21]
I noticed there are some settings in the menu that are not explained in the documentation. Under Tracker Settings there are 'Active Observation', 'Multi-Anchor Placement', 'Single-Anchor Placement' and 'Single-Anchor Scaled'. What do these do?

**Comment by craigktreasure** [comment]
Good call. We'll try to get this information updated in our documentation in our next round of updates.

Internal reference number is 38627534.
